pymeasure/instruments/attocube/anc300.py

```diff
@@ -129,17 +135,40 @@ class Axis(Channel):
         "getc",
         """Measure the saved capacity value in nF of the axis.""")
 
+    continuous_move = Instrument.setting(
+        "%s",
+        """Set a continous movement in 'up' or 'down' direction. Mode must be
+        'stp'. The axis can be halted by the stop method.
+        """,
+        validator=strict_discrete_set,
+        values={"up": "stepu c", "down": "stepd c"},
+        map_values=True,
+        check_set_errors=True,
+    )
+
     stepu = Instrument.setting(
         "stepu %d",
         """Set the steps upwards for N steps. Mode must be 'stp' and N must be
-        positive.""",
-        validator=strict_range, values=[0, inf], check_set_errors=True)
+        positive. 0 causes a continous movement until stop is called.
+
+        .. deprecated:: Use attr:`continuous_move` or meth:`move_raw` instead.
```

Comment:
Please add the version, since when it is deprecated, see https://github.com/pymeasure/pymeasure/issues/672#issuecomment-1371900222
```suggestion
        .. deprecated:: 0.13.0 Use attr:`continuous_move` or meth:`move_raw` instead.
```

Reply:
implemented with the other changes below